internal/commands/internal/test/test_test.go

```diff
@@ -32,3 +33,23 @@ func TestSimple(t *testing.T) {
 	adaptedPath := filepath.FromSlash("dir/subdir")
 	assert.Contains(t, buf.String(), fmt.Sprintf("'%s' succeeded", adaptedPath))
 }
+
+func TestDryRunAll(t *testing.T) {
```

Comment:
This is a bit confusing: any `cirrus internal test` invocation is a dry run.

Why not just test the specific little thing, and abstract away the boilerplate if needed?

Reply:
All the tests cases so far are valid and expected to succeed. This test just dry runs all of the them to make sure they succeed without extra checks like other tests (checking logs, etc.).

Can rename to `TestAll` 🤷‍♂️

Reply:
That (or at least a comment that clarifies why it's "dry") would be better.

Reply:
See e756614